fcrepo-http-commons/src/test/java/org/fcrepo/http/commons/domain/PreferTagTest.java

```diff
@@ -32,4 +35,15 @@ public void testEmpty() {
         assertEquals("", preferTag.getValue());
         assertTrue(preferTag.getParams().isEmpty());
     }
+
```

Comment:
I followed the https://wiki.duraspace.org/display/FF/Git+Workflow

$ git commit --amend .
$ git push -f origin fcrepo-1234

So what you said was changed to
$ git commit --amend .
$ git push origin fcrepo-1234
?



On Thu, Jan 29, 2015 at 12:11 PM, Andrew Woods <notifications@github.com>
wrote:

> In
> fcrepo-http-commons/src/test/java/org/fcrepo/http/commons/domain/PreferTagTest.java
> <https://github.com/fcrepo4/fcrepo4/pull/699#discussion_r23783914>:
>
> > @@ -32,4 +35,15 @@ public void testEmpty() {
> >          assertEquals("", preferTag.getValue());
> >          assertTrue(preferTag.getParams().isEmpty());
> >      }
> > +
>
> It looks like you may have done a force-push. In the future, please add
> subsequent commits to the same branch instead of mashing updates into a
> single commit. That way it is easier to review and see your follow-on
> updates.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fcrepo4/fcrepo4/pull/699/files#r23783914>.
>